ui/app/scripts/app/views/register-modules.html

```diff
@@ -0,0 +1,64 @@
+<h1>Register Applications</h1>
+<p>Register one or more applications by entering <strong>Name</strong>, <strong>Type</strong> and <strong>URI</strong> of the application JAR. The URI is typically Maven coordinates of the JAR, but can be local file or docker image URI.</p>
```

Comment:
Wording:
Register one or more applications by entering *a* Name, Type and URI of the application *Jar*. The URI is typically *provided using the* Maven coordinates of the *Jar*, but can be a local file or docker image URI.

We should also provide at least an example for the Maven coordinates, e.g.:

`maven://io.spring.cloud:scdf-batch-simple:jar:exec:1.0.0.BUILD-SNAPSHOT`